src/main/java/com/sap/oss/phosphor/fosstars/model/value/ArtifactVersionValue.java

```diff
@@ -0,0 +1,69 @@
+package com.sap.oss.phosphor.fosstars.model.value;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import java.util.Objects;
+
+/**
+ * A value for a feature that provide a version for an artifact.
+ */
+public class ArtifactVersionValue extends AbstractValue<String> {
```

Comment:
Yes, you are right.
Current assumption is that an open source project may provide different artifacts but all with the same versions.
One option would to add the artifact name in form of a package url (Purl).
Either as additional field to `ArtifactVersion` and change here to `AbstractValue <ArtifactVersion>`.
Only disadvantage I see is that current `ArtifactVersion` requires a release date, but if `ArtifactVersion` is used for `ArtifactVersionValue` the release data must be optional.
Or we add an `ArtifactVersionWithReleaseDate` 😉

Reply:
Created #478 for refactor the Artifact / ArtifactVersion

Reply:
In this pull request, let's please rename `ArtifactVersionValue` to `ArtifactVersionStringValue` and `ArtifactVersionFeature` to `ArtifactVersionStringFeature` since they are about versions strings.